src/admin/models/user.ts

```diff
@@ -8,6 +8,8 @@ export interface User {
   gender: string;
   state: string;
   phoneNumber: string;
+  serverHost: string;
```

Comment:
@MurhafSousli  not sure server host should be inside the user interface. It is more an application setting